From c105b16d78de778fcd47722c71274a5a6a10c6d5 Mon Sep 17 00:00:00 2001
From: Dan Callaghan <dcallagh@redhat.com>
Date: Wed, 11 Jul 2018 12:48:51 +1000
Subject: [PATCH 06/10] Set powerpc logo hint

Previously, when the CPU information was coming from /proc/cpuinfo, this
logo hint was being set.

---
 src/core/device-tree.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/core/device-tree.cc b/src/core/device-tree.cc
index 642b3c5..7903622 100644
--- a/src/core/device-tree.cc
+++ b/src/core/device-tree.cc
@@ -695,6 +695,7 @@ static void scan_devtree_cpu_power(hwNode & core)
     }
 
     cpu.setDescription("CPU");
+    cpu.addHint("logo", string("powerpc"));
     set_cpu(cpu, currentcpu++, basepath);
 
     reg = get_u32(basepath + "/reg");
-- 
2.17.1

